### ISSUE TITLE
Twitter Bot - Customized

### ISSUE THREAD

**Issue by ch3nkula** [Ch3ck/Cameroon#1]
You can use this since it's customized already (rather than reinventing the wheel) :). https://github.com/ch3nkula/Wiki-Twit-Bot. It's customized and you can just put your preferred hash-tags in the text file `hash-tags.txt` and deploy. It's in Node JS except you want to build a different bot in a different technology. Thanks @Ch3ck.

**Comment by Ch3ck** [comment]
Yeah, i"m looking at a Go or Python implementation for this @ch3nkula I can't understand JS yet.
